pyproject.toml

```diff
@@ -0,0 +1,6 @@
+[tool.black]
+line-length = 120
+
+exclude = """
+    .
```

Comment:
(ignore all) TEMP

Reply:
Lets black everything in this PR?

Reply:
Mmmmm,
I can do so but maybe we shall wait for other PRs to be closed? So we will avoid conflicts.
(I'm suggesting to black everything in a different consecutive PR 🙂 )